scripts/add-or-update-pr-issue-templates.sh

```diff
@@ -0,0 +1,165 @@
+-->
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+EOF
+}
+
+function create_pr_template {
+cat << "EOF" > .github/pull_request_template.md
+<!--
+  Have any questions? Check out the contributing docs at https://docs.gruntwork.io/guides/contributing/, or
+  ask in this Pull Request and a Gruntwork core maintainer will be happy to help :)
+  Note: Remember to add '[WIP]' to the beginning of the title if this PR is still a work-in-progress.
+-->
+
+## Description
+
+<!-- Write a brief description of the changes introduced by this PR -->
+
+### Documentation
+
+<!--
+  If this is a feature PR, then where is it documented?
+
+  - If docs exist:
+    - Update any references, if relevant.
+  - If no docs exist:
+    - Create a stub for documentation including bullet points for how to use the feature, code snippets (including from happy path tests), etc.
+-->
+
+<!-- Important: Did you make any backwards incompatible changes? If yes, then you must write a migration guide! -->
+
+## TODOs
+
+- [ ] Ensure the branch is named correctly with the issue number. e.g: `feature/new-vpc-endpoints-955` or `bug/missing-count-param-434`.
+- [ ] Update the docs.
+- [ ] Keep the changes backwards compatible where possible.
+- [ ] Run the pre-commit checks successfully.
+- [ ] Run the relevant tests successfully.
+- [ ] Ensure any 3rd party code adheres with our license policy: https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378
+- [ ] _Maintainers Only._ If necessary, release a new version of this repo.
+- [ ] _Maintainers Only._ If there were backwards incompatible changes, include a migration guide in the release notes.
+- [ ] _Maintainers Only._ Add to the next version of the monthly newsletter (see https://www.notion.so/gruntwork/Monthly-Newsletter-9198cbe7f8914d4abce23dca7b435f43).
+
+
+## Related Issues
+
+<!--
+  Link to the issue that is fixed by this PR (if there is one)
+  e.g. Fixes #1234
+
+  Link to an issue that is partially addressed by this PR (if there are any)
+  e.g. Addresses #1234
+
+  Link to related issues (if there are any)
+  e.g. Related to #1234
+-->
+EOF
+}
+
+# Ensure the GitHub template directories exist
+mkdir -p .github
+mkdir -p .github/ISSUE_TEMPLATE
+
+# if the repo does not contain a bug_report.md file, then create one or replace the existing one
+if [ ! -f ".github/ISSUE_TEMPLATE/bug_report.md" ]; then 
+	echo "Could not find file at .github/ISSUE_TEMPLATE/bug_report.md, so adding one..."
+	create_bug_issue_template
+else
+    echo "Found file at .github/ISSUE_TEMPLATE/bug_report.md, so replacing it..."
+    rm .github/ISSUE_TEMPLATE/bug_report.md
+	create_bug_issue_template
```

Comment:
NIT: indentation issue?

Reply:
Seems like odd indentation throughout the file. Check your editor settings!

Reply:
@brikis98 something is definitely fishy. Thanks for spotting this!